### PR TITLE
AI Assistant: Add keyboard shortcuts

### LINKED ISSUE #30935: AI Assistant stop button: add keyboard shortcut or move button to fixed position
It can get tricky to click it since the button is moving while the response is being received.

https://user-images.githubusercontent.com/16329583/240893992-eb150d5c-b9fa-4529-bcf3-2821971b660a.mov

### PR DESCRIPTION
**Pull request by renatoagds**
Add shortcuts to `Stop` and `Accept` and improved `Send` shortcut.

Closes #30935

## Proposed changes:

* Add new shortcut to `Stop` and `Accept`.
* Add new layout to show shortcuts for user.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/A

## Does this pull request change what data or activity we track or use?

N/A

## Testing instructions:

* Add the block.
* Ask for something and send using `Enter`.
* Stop it using `Esc`.
* Confirm using `CMD/Control+Enter`.

## Demo

https://github.com/Automattic/jetpack/assets/1663717/8925f8d4-5cc7-41a9-b0cd-007992c80b2a

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js [hunk 1]
```diff
@@ -170,54 +190,68 @@ const AIControl = forwardRef(
...
-							>
-								<Icon icon={ closeSmall } />
-								{ __( 'Stop', 'jetpack' ) }
-							</Button>
-						) }
-
-						{ contentIsLoaded &&
-							! isWaitingState &&
-							( isInBlockEditor && promptType === 'generateTitle' ? (
+						<div className="jetpack-ai-assistant__prompt_button_wrapper">
```
Comment (renatoagds) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js:200]: The diff got crazy here, but no change related with the buttons, just wrapped it in this new `div` to add the `keyboard` hint.

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/edit.js [hunk 2]
```diff
@@ -198,7 +220,7 @@ export default function AIAssistantEdit( { attributes, setAttributes, clientId }
 				ref={ aiControlRef }
 				content={ attributes.content }
 				contentIsLoaded={ contentIsLoaded }
-				getSuggestionFromOpenAI={ getSuggestionFromOpenAI }
+				getSuggestionFromOpenAI={ handleGetSuggestion }
```
Comment (renatoagds) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/edit.js:229]: It stays the same, we just go through the `handleGetSuggestion` to make sure we focus the block, to allow the esc to happen.

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js [hunk 3]
```diff
@@ -97,6 +92,32 @@ const AIControl = forwardRef(
 			[]
 		);
 
+		useKeyboardShortcut(
+			[ 'command+enter', 'ctrl+enter' ],
+			() => {
+				if ( contentIsLoaded ) {
+					if ( promptType === 'generateTitle' ) {
+						handleAcceptTitle();
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js:100]: This needs a check like this one:
https://github.com/Automattic/jetpack/pull/31255/files#diff-37665ae34bb3f7c5fd67134163c60c9a91d6207a446fb0dec8db6616cd2128eeR200

Maybe the check can go inside the `handleAcceptTitle` function just to be a bit more future-proof.

File: projects/plugins/jetpack/extensions/blocks/ai-assistant/editor.scss [hunk 4]
```diff
@@ -138,6 +144,16 @@
 			cursor: not-allowed;
 		}
 	}
+
+	.jetpack-ai-assistant__prompt_button_keyboard {
+		opacity: 0.6;
+		font-size: 8px;
+		background: var(--wp--preset--color--cyan-bluish-gray);
+		color: var(--wp--preset--color--base);
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/ai-assistant/editor.scss]:
```suggestion
		background: var( --wp--preset--color--cyan-bluish-gray );
		color: var( --wp--preset--color--base );
```

**Review by dhasilva — DISMISSED** [review]
LGTM!

**Review by dhasilva — APPROVED** [review]

**Pull request merged**